main.tf

```diff
@@ -4,7 +4,7 @@ locals {
 }
 
 resource "aws_dynamodb_table" "this" {
-  count = var.create_table && !var.autoscaling_enabled ? 1 : 0
+  count = module.this.enabled ? var.create_table && !var.autoscaling_enabled ? 1 : 0 : 0
 
   name                        = var.name
```

Comment:
are you sure this does not need to be adjusted due to you now integrating the label/context module?